src/registries/menus/topbar_menu_registry.ts

```diff
@@ -399,6 +399,15 @@ topbarMenuRegistry
     ...ACTION_FORMAT.formatWrappingClip,
     sequence: 30,
   })
+  .addChild("format_increase_indent", ["format"], {
+    ...ACTION_FORMAT.styleIncreaseIndent,
```

Comment:
I'd swap the entries, I feel like the button to indent (translate text to the right) shoudlbe at the right (& vice versa)